### PR TITLE
Add: ks-nav tool set

### PR DESCRIPTION
**Pull request by alessandrocarminati**
ks-nav is a set of two tools developed to support ELISA activities.

Signed-off-by: Alessandro Carminati <acarmina@redhat.com>
Signed-off-by: Maurizio Papini <mpapini@redhat.com>

### CODE REVIEW

File: ks-nav/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Contributing to nav
+
+:+1::tada: First off, thanks for taking the time to 
+contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to nav.
+The nav utility is hosted in the 
+[nav repository](https://github.com/alessandrocarminati/nav)
```
Comment (reiterative) [inline ks-nav/CONTRIBUTING.md:8]: Should this refer to the ELISA GitHub repo instead?

File: ks-nav/kern_bin_db/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Contributing to kern_bin_db
+
+:+1::tada: First off, thanks for taking the time to 
+contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to kern_bin_db.
+The kern_bin_db utility is hosted in the 
+[kern_bin_db repository](https://github.com/alessandrocarminati/kern_bin_db-)
```
Comment (reiterative) [inline ks-nav/kern_bin_db/CONTRIBUTING.md]: Should this refer to the ELISA GitHub repo instead?

**Comment by reiterative** [comment]
Are the MAINTAINERS files intended to be included in this repository, or should they be provided by the user as input for a specific analysis?

**Comment by alessandrocarminati** [comment]
@reiterative  The "MAINTAINERS" file is specific to the kernel version under analysis. It is a living object that changes accordingly with changes in the kernel code base. 
The tool that analyse the kernel image is supposed to access to the proper "MAINTAINERS" file during the analysis to be able to provide coherent data.

File: ks-nav/CONTRIBUTING.md (hunk 1 above)
Comment (alessandrocarminati) [reply ks-nav/CONTRIBUTING.md:8]: Currently, I am developing the tool, which is not yet finished in the repository that is listed in the README file. The link's intent was to provide the source where a user could find the newest version. If required, I can change them instantly.

File: ks-nav/kern_bin_db/CONTRIBUTING.md (hunk 2 above)
Comment (alessandrocarminati) [reply ks-nav/kern_bin_db/CONTRIBUTING.md]: Same as above.

File: ks-nav/CONTRIBUTING.md (hunk 1 above)
Comment (paolonig) [reply ks-nav/CONTRIBUTING.md:8]: I think we should move everything to the ELISA github and remove personal repos

File: ks-nav/kern_bin_db/CONTRIBUTING.md (hunk 2 above)
Comment (paolonig) [reply ks-nav/kern_bin_db/CONTRIBUTING.md]: Same as above

File: ks-nav/CONTRIBUTING.md (hunk 1 above)
Comment (paolonig) [reply ks-nav/CONTRIBUTING.md:8]: This has been solved in pull request 12

File: ks-nav/kern_bin_db/CONTRIBUTING.md (hunk 2 above)
Comment (paolonig) [reply ks-nav/kern_bin_db/CONTRIBUTING.md]: This has been solved in pull request 12

**Comment by paolonig** [comment]
This pull request is superseded by pull #12

**Pull request closed without merging**